.travis.yml

```diff
@@ -30,6 +30,13 @@ matrix:
         - TEST_COMMAND="test:compile"
       scala:
         - 2.11.12
+    - os: linux
```

Comment:
i don't think i want to add another CI matrix for this as it does slow down build times with an entire `scalac` compile and `jvm` warmup. Especially for something as small as this. 

I'm not sure if we should test this on CI everytime.